base_rest/__manifest__.py

```diff
@@ -6,18 +6,26 @@
     "summary": """
         Develop your own high level REST APIs for Odoo thanks to this addon.
         """,
-    "version": "14.0.4.3.1",
+    "version": "15.0.1.0.0",
     "development_status": "Beta",
     "license": "LGPL-3",
     "author": "ACSONE SA/NV, " "Odoo Community Association (OCA)",
     "maintainers": ["lmignon"],
     "website": "https://github.com/OCA/rest-framework",
     "depends": ["component"],
     "data": [
-        "views/assets_template.xml",
```

Comment:
Why do you remove this file? Without the assets, the openapi UI will no more work.

Reply:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0
```
Assets are now directly declared in the manifest. You should move .js and .scss links from templates to __manifest__.py file.
```

Reply:
But the problem isn't with python code in checks. I can't understand what is wrong with README and index. Codecov check fails on that.

Reply:
OK my bad. Thank you for the link.

Reply:
you're welcome